src/components/Modals/PlaceBetModal.tsx

```diff
@@ -30,9 +30,7 @@ export const PlaceBetModal: React.FC<Props> = ({
   const [txLoading, setTxLoading] = useState(false);
   const [txHash, setTxHash] = useState<string>();
   const [error, setError] = useState<ethers.errors>();
-
```

Comment:
the line separations are here for readability - i dont think they should be removed